js/apps/account-ui/src/personal-info/FormField.tsx

```diff
@@ -25,6 +26,7 @@ export const FormField = ({ attribute }: FormFieldProps) => {
   const isSelect = (attribute: UserProfileAttributeMetadata) =>
     Object.hasOwn(attribute.validators, "options");
 
+  if (attribute.name === "locale") return <LocaleSelector />;
```

Comment:
I don't think the `FormField` component should be the one responsible for this. This should be handled higher up in the component hierarchy. This should be generic component that can be re-used, without any side-effects.

Reply:
Why it's part of the user profile it's just one of the attributes